.github/workflows/daily-platform-build.yml

```diff
@@ -0,0 +1,78 @@
+name: Daily Platform Build
+
+on: [pull_request, workflow_dispatch]
+
+jobs:
+  artifacts_build:
+    name: "Build artifacts"
+    runs-on: [self-hosted, Linux, X64, Diff, Ubuntu20.04]
+    env:
+      THREADS: 24
+      PY_VERSION: "3.9"
+
+    steps:
+      - name: Set up repository
+        uses: actions/checkout@v3
+        with:
+          # Number of commits to fetch. `0` indicates all history for all
+          # branches and tags. (default: 1)
+          fetch-depth: 0
+          token: ${{ secrets.PAT }}
+
+      - name: Build community binaries
+        run: |
+          git clone -b master https://github.com/memgraph/memgraph.git
+
+          cd memgraph
+
+          # Activate toolchain.
+          source /opt/toolchain-v4/activate
+
+          # Initialize dependencies.
+          ./init
+
+          # Build community binaries.
+          cd build
+          cmake -DCMAKE_BUILD_TYPE=release -DMG_ENTERPRISE=OFF ..
+          make -j$THREADS && make -j$THREADS mgconsole
+
+          mkdir -p output
+          cd output
+
+          cpack -G DEB --config "../CPackConfig.cmake"
+
+          cd ../../../
+
+          mkdir -p scripts/resources
+
+          find memgraph/build/output -type f -name "*.deb" -exec mv -f {} scripts/resources/memgraph-daily-deb_amd64.deb \;
+
+          rm -rf memgraph
+
+      - name: "Upload DEB"
+        uses: actions/upload-artifact@v3
+        with:
+          name: daily-deb
+          path: scripts/resources/memgraph*.deb
+
+      # - name: Build MAGE image
+      #   run: |
+      #     cd scripts
+
+      #     ./build_mage_img.sh main daily-mage daily-deb
+
+      # - name: "Upload MAGE image"
+      #   uses: actions/upload-artifact@v3
+      #   with:
+      #     name: daily-mage
+      #     path: resources/daily-mage.tar.gz
+
+      # - name: Build platform image
+      #   run: |
+      #     ./build_platform_img.sh master daily-platform daily-deb ${{ secrets.PAT }}
+
+      # - name: "Upload platform image"
+      #   uses: actions/upload-artifact@v3
+      #   with:
+      #     name: daily-platform
+      #     path: resources/daily-platform.tar.gz
```

Comment:
To keep this or remove the comment?

Reply:
I am creating a new workflow, there I removed the comment and will fill the gap with new steps as I go along.